macstadium-shared-1/Makefile

```diff
@@ -18,51 +36,51 @@ $(CONFIG_FILES):
-	trvs generate-config -p TRAVIS_WORKER -f env macstadium-workers custom-2 \
+	trvs generate-config -n -p TRAVIS_WORKER -f env macstadium-workers custom-2 \
 		| sed 's/^/export /' >config/travis-worker-custom-2
-	trvs generate-config -p TRAVIS_WORKER -f env macstadium-workers custom-3 \
+	trvs generate-config -n -p TRAVIS_WORKER -f env macstadium-workers custom-3 \
 		| sed 's/^/export /' >config/travis-worker-custom-3
 	trvs generate-config --pro -p VSPHERE_JANITOR -f env vsphere-janitor production-1 \
 		| sed 's/^/export /' >config/vsphere-janitor-prod-com
-	trvs generate-config --pro -p VSPHERE_JANITOR -f env vsphere-janitor staging-1 \
+	trvs generate-config -n --pro -p VSPHERE_JANITOR -f env vsphere-janitor staging-1 \
 		| sed 's/^/export /' >config/vsphere-janitor-staging-com
-	trvs generate-config -p VSPHERE_JANITOR -f env vsphere-janitor custom-1 \
+	trvs generate-config -n -p VSPHERE_JANITOR -f env vsphere-janitor custom-1 \
 		| sed 's/^/export /' >config/vsphere-janitor-custom-1
-	trvs generate-config -p VSPHERE_JANITOR -f env vsphere-janitor custom-2 \
+	trvs generate-config -n -p VSPHERE_JANITOR -f env vsphere-janitor custom-2 \
 		| sed 's/^/export /' >config/vsphere-janitor-custom-2
-	trvs generate-config -p VSPHERE_JANITOR -f env vsphere-janitor custom-3 \
+	trvs generate-config -n -p VSPHERE_JANITOR -f env vsphere-janitor custom-3 \
 		| sed 's/^/export /' >config/vsphere-janitor-custom-3
-	trvs generate-config --pro -p JUPITER_BRAIN -f env jupiter-brain production-1 \
+	trvs generate-config -n --pro -p JUPITER_BRAIN -f env jupiter-brain production-1 \
 		| sed 's/^/export /' >config/jupiter-brain-prod-com-env
-	trvs generate-config --pro -p JUPITER_BRAIN -f env jupiter-brain staging-1 \
+	trvs generate-config -n --pro -p JUPITER_BRAIN -f env jupiter-brain staging-1 \
 		| sed 's/^/export /' >config/jupiter-brain-staging-com-env
-	trvs generate-config --pro -p TRAVIS_WORKER -f env macstadium-workers staging-1-1 \
+	trvs generate-config -n --pro -p TRAVIS_WORKER -f env macstadium-workers staging-1-1 \
 		| sed 's/^/export /' >config/travis-worker-com-staging-1
-	trvs generate-config --pro -p TRAVIS_WORKER -f env macstadium-workers staging-1-2 \
+	trvs generate-config -n --pro -p TRAVIS_WORKER -f env macstadium-workers staging-1-2 \
 		| sed 's/^/export /' >config/travis-worker-com-staging-2
-	trvs generate-config --pro -p TRAVIS_WORKER -f env macstadium-workers production-1-1 \
+	trvs generate-config -n --pro -p TRAVIS_WORKER -f env macstadium-workers production-1-1 \
 		| sed 's/^/export /' >config/travis-worker-com-prod-1
-	trvs generate-config --pro -p TRAVIS_WORKER -f env macstadium-workers production-1-2 \
+	trvs generate-config -n --pro -p TRAVIS_WORKER -f env macstadium-workers production-1-2 \
 		| sed 's/^/export /' >config/travis-worker-com-prod-2
-	trvs generate-config -p COLLECTD_VSPHERE -f env collectd-vsphere common-1 \
+	trvs generate-config -n -p COLLECTD_VSPHERE -f env collectd-vsphere common-1 \
```

Comment:
I think that's good for two reasons:

1. It's faster because it's fewer `git pulls`
1. It makes sure that we only have one commit of keychain for all the configs, instead of currently when it's possible that someone pushes something in the middle of generating all the configs, and we'll have one version for some commits and another for other configs.

(Also, there's nothing ensuring that we're getting the `master` branch, this still runs fine if you have a branch of keychain checked out, it just makes sure that you have everything that's been pushed to GitHub. You don't even need to push your latest changes, as long as it's committed and the branch exists on the remote)